CONTRIBUTING.md

```diff
@@ -157,9 +157,9 @@ $ cat foobar.json
 }
 ```
 
-And we would just call colin and point it to `/tmp/external_checks/checks.py`:
+And we would just call colin and point it to the directory containing `/tmp/external_checks/checks.py`:
```

Comment:
I am not sure if we now need the file example. What about removing it since we have an example bellow?

Reply:
I only see one example related to calling colin on external checks?

Reply:
I mean that we can remove the example path above since we have full example bellow.

Something like this:
```suggestion
And we would just call colin and point it to the directory containing python files with checks:
```

Reply:
Sounds good, will update my commit.